tests/test_quantities.py

```diff
@@ -0,0 +1,147 @@
+"""Tests for quantities module functions."""
+
+import pytest
+import torch
+from numpy.testing import assert_allclose
+
+from torch_sim.quantities import calc_heat_flux
+
+
+class TestHeatFlux:
+    """Test suite for heat flux calculations."""
+
+    @pytest.fixture
+    def mock_simple_system(
+        self,
+        device: torch.device,
+    ) -> dict[str, torch.Tensor]:
+        """Simple system with known values."""
+        return {
+            "velocities": torch.tensor(
+                [
+                    [1.0, 0.0, 0.0],
+                    [0.0, 2.0, 0.0],
+                    [0.0, 0.0, 3.0],
+                ],
+                device=device,
+            ),
+            "energies": torch.tensor([1.0, 2.0, 3.0], device=device),
+            "stress": torch.tensor(
+                [
+                    [1.0, 0.0, 0.0, 0.0, 0.0, 0.0],
+                    [0.0, 2.0, 0.0, 0.0, 0.0, 0.0],
+                    [0.0, 0.0, 3.0, 0.0, 0.0, 0.0],
+                ],
+                device=device,
+            ),
+            "masses": torch.ones(3, device=device),
+        }
+
+    def test_unbatched_total_flux(
+        self, mock_simple_system: dict[str, torch.Tensor]
+    ) -> None:
+        """Test total heat flux calculation for unbatched case."""
+        flux = calc_heat_flux(
+            momenta=None,
+            masses=mock_simple_system["masses"],
+            velocities=mock_simple_system["velocities"],
+            energies=mock_simple_system["energies"],
+            stress=mock_simple_system["stress"],
+            is_virial_only=False,
+        )
+
+        # Heat flux parts should cancel out
+        expected = torch.zeros(3, device=flux.device)
+        assert_allclose(flux.cpu().numpy(), expected.cpu().numpy())
+
+    def test_unbatched_virial_only(
+        self, mock_simple_system: dict[str, torch.Tensor]
+    ) -> None:
+        """Test virial-only heat flux calculation for unbatched case."""
+        virial = calc_heat_flux(
+            momenta=None,
+            masses=mock_simple_system["masses"],
+            velocities=mock_simple_system["velocities"],
+            energies=mock_simple_system["energies"],
+            stress=mock_simple_system["stress"],
+            is_virial_only=True,
+        )
+
+        expected = -torch.tensor([1.0, 4.0, 9.0], device=virial.device)
+        assert_allclose(virial.cpu().numpy(), expected.cpu().numpy())
+
+    def test_batched_calculation(self, device: torch.device) -> None:
```

Comment:
Could you add a test to check that if I run `unbatched(A)` and `unbatched(B)` I get the same results as `batched(A, B)`?